app/controllers/callbacks_controller.rb

```diff
@@ -1,11 +1,22 @@
 # frozen_string_literal: true
 
 class CallbacksController < Devise::OmniauthCallbacksController
+  WHITELISTED_ORGS = ENV['ORGANIZATIONS'].split(',').map(&:downcase).freeze
+
   def github
-    user = User.from_omniauth(request.env['omniauth.auth'])
-    sign_in_and_redirect user
+    auth = request.env['omniauth.auth']
```

Comment:
I believe the easiest way to simplify the code here is to get rid of explicit `auth` variable, and just inline it.
I also suppose it should be GC-friendlier, since inlined evaluated parameter is clearly a short-lived object (and a specially-allocated variable — not necessary)